dapp_runner/runner/runner.py

```diff
@@ -235,7 +235,11 @@ def dapp_state(self) -> Dict[str, Dict[int, ServiceState]]:
                 instance_index: instance.state
                 for instance_index, instance in enumerate(cluster.instances)
             }
-            for cluster_id, cluster in self.clusters.items()
+            if cluster
```

Comment:
Comprehensions here are way to complex for my taste. Going back to something with more lines would be needed. Also we have unused variables and shadowing.

Reply:
@approxit simplified the construction of the final dictionary, please re-review